projects/bs4/project.yaml

```diff
@@ -0,0 +1,12 @@
+homepage: "https://www.crummy.com/software/BeautifulSoup/"
+main_repo: "https://code.launchpad.net/~leonardr/beautifulsoup/bs4"
+language: python
+primary_contact: "security-tps@google.com"
```

Comment:
need to remember to get dev contact .

Reply:
Since I'll be receiving the alerts, I won't forget :)